### DISCUSSION TITLE
Action steps after failed website relaunch

### DISCUSSION THREAD

**Discussion by DanM3rcurius** [SecretDecks/Documentation#106] [Ideas]
# Retrospective of Website relaunch
- What went wrong:  
  - no planning/estimation of work; rushed release date; 
  - no project "caretaker" (shared responsibility not yet working); 
  - low SD-TownHall participation of core members; 
  - communication between members not aligned with Mindfulness Policy ("Solutioning, over discussing problems") and breaking CoC

- What went right:
  - founders drop page was live for testing
  - established workflow v.01 from concept to website 
  - 

# Proposed action items
###  1. Signing off on mindfulness policy and other OS docs 
- how contribute, Code of Conduct
-> mitigation of miscommunications & facilitation of solution oriented discussions

### 2. Commitment from core
- participation in at least 75% of weeklies, monthly filing of contribution survey, monthly peer approval of contribution surveys, participation in 80% of retrospectives
-> to form a strong and resilient core

### 3. Contribution Survey has to be filed within 90 days 
- starting from last day of the respective month
-> to have a clear an timely overview of expenses and be transparent within the core and to the outside

### 4. Finer granulation of type of work (or even coop "spheres"):

- CORE contributor ("sphere") 
  - (post paid if funds available; freelancer & caretakers inside the core cooperative "sphere"/"orbit", then access to gov token (full voting & polling rights), exclusive NFTs, reputation NFT

- (SHELL) pro-bono work ("sphere")
  - (post-paid if funds available); freelancer outside the cooperative or in the SHELL "sphere"/"orbit" then access to *polling and basic voting* rights 

- (SHELL) livelihood ("sphere")
  - (paid per task or hour upon completion); freelancer or companies / services outside the cooperative or in the SHELL "sphere"/"orbit", then access to *polling* rights)

### 5. Project sized tasks (like a website relaunch or the founders drop) need to have at least two "caretakers" who take responsibility for:  
 - facilitating kickoff, estimation, "sprint" and retrospective meetings, tracking progress, creating reports for SD-TownHall, 
Taking this responsibility and delivering successfully will give a bonus (multiplier of (?) applied to total (ADA) in contribution survey

**Comment by DanM3rcurius** [comment]
@SecretDecks/founding-core 
Please provide input, this will be on the agenda of tomorrows TownHall.

**Reply by wlstylr** [reply]
Thanks for drafting this :) 

I just want to touch on point # 5.
Ultimately, I think we tried to do too much, too quickly without great planning.  However, I think it was cool to see how we all were able to create so much content from wireframes, to live web pages, to animations, written content etc 💪 

In the future, I think adding "sprint planning" into our Tuesday weekly meetings, and sprint review and retro to Friday, or another day after that works for us all would help the process.   Sprint review can include having a core vote to decide whether or not to release what was accomplished during the sprint for the week.  We can use retro as an opportunity to share how we think the week went and identify a way in which we can improve the next sprint.

**Reply by DanM3rcurius** [reply]
> Thanks for drafting this :)
> 
> I just want to touch on point # 5.
> Ultimately, I think we tried to do too much, too quickly without great planning. However, I think it was cool to see how we all were able to create so much content from wireframes, to live web pages, to animations, written content etc 💪

Word! 

> 
> In the future, I think adding "sprint planning" into our Tuesday weekly meetings, and sprint review and retro to Friday, or another day after that works for us all would help the process. Sprint review can include having a core vote to decide whether or not to release what was accomplished during the sprint for the week. We can use retro as an opportunity to share how we think the week went and identify a way in which we can improve the next sprint.

Very good idea! This will mean we have to have commitment from core members, to allow voting on such a high impact level.

**Comment by stephen-rowan** [comment]
## Introducing Agile to GitHub

For a while in some of my QA-DAO repos I have been experimenting with using the ZenHub integration ( https://www.zenhub.com/ ). This allows you to add Agile techniques to GitHub project boards (e.g. sprints etc) and also a roadmap view (screenshots below). I think it is free for small teams. If SecretDecks approves maybe this approach could be a QA-DAO collaboration in preparation for my Open-Source Training proposal ?

## Sprints
![2021-09-28](https://user-images.githubusercontent.com/25156451/135089545-03479924-8a3d-4d6d-9bad-7c75b0331da1.png)
## Roadmap 
![2021-09-28 (1)](https://user-images.githubusercontent.com/25156451/135089767-3509c288-5725-407b-877c-b9f3eae216ad.png)

**Comment by stephen-rowan** [comment]
## Improving the Contribution Survey

From today, I am piloting Issue automation on Swarms "Hive" repo - raising a daily "Core-Swarm-Activities" issue through a GitHub Action. The YAML looks like [this](https://github.com/Catalyst-Swarm/Catalyst-Beehive/blob/main/.github/workflows/Core-Swarm-Activities.yml) and the Issue Bot like [this](https://github.com/Ek-phrasis/Sandbox/issues/32)  . This action closes the previous issue and opens a new one for that day.

To streamline SD's contribution survey I want to build an action that -
- raises a contribution survey at the end of each month
- closes the contribution survey for the previous month
- allocates the contribution survey to each core member and prefills their details
- a more advanced version could look up the profile of a core member and only ask relevant contribution questions
- uses a Issue Form template with dropdowns (no long lists of checkboxes)
- sends notification bots to Discord
- a more advanced step would process the data collected into a csv etc.

All this will take time - but is pretty straightforward. I will start with a simple monthly automation.

**Comment by DanM3rcurius** [comment]
Yes Yes and Yes please!

**Comment by DanM3rcurius** [comment]
This is terrific! Thank you Stephen!
I’ll shave of an hour from the next mornings and get into YAML. Should we create a separate project for this?
